db/migrate/20200521153435_remove_pro_memberships.rb

```diff
@@ -0,0 +1,21 @@
+class RemoveProMemberships < ActiveRecord::Migration[5.2]
+  def up
```

Comment:
FYI: `drop_table` in `change` accepts a block that is used for the creation step: https://github.com/thepracticaldev/dev.to/blob/master/db/migrate/20191227113154_drop_push_notification_subscriptions.rb

Reply:
oooh that is great to know! Thank you!